config/default.json

```diff
@@ -2,6 +2,7 @@
 	"scope": "springernature",
 	"packagesDirectory": "./packages",
 	"changelog": "HISTORY.md",
+	"CSSDirectoryStructure": ["10-settings", "20-functions", "30-mixins", "40-base", "50-components", "60-utilities"],
 	"allowExtends": false,
```

Comment:
this should not be in the default config. we don't specify any folder structure by default. So you need to remove this and allow it to be configured by each of the mono-repos. You can add this to the test configuration to test this.